maven/Dockerfile

```diff
@@ -1,6 +1,30 @@
 FROM ghcr.io/dependabot/dependabot-updater-core
 
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    openjdk-21-jdk \
+    # avoids keytool usage
+    ca-certificates-java \
+    # we need to allow the dependabot user to write to these files for when update-ca-certificates is run
+    && chgrp dependabot /etc/default/cacerts  \
+    && chmod g+rw /etc/default/cacerts \
+    && chgrp dependabot /etc/ssl/certs/java/cacerts  \
+    && chmod g+rw /etc/ssl/certs/java/cacerts \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY --from=maven:3.9.9 /usr/share/maven /usr/share/maven
```

Comment:
super-nit: they released `3.9.10` (just to spite you 😄)

https://github.com/apache/maven/releases/tag/maven-3.9.10

Reply:
[3.9.10 container image hasn't been released yet](https://hub.docker.com/_/maven/tags?name=3.9.10), so I'll keep 3.9.9 for now :)

Reply:
A newer version is out now :) 

https://github.com/dependabot/dependabot-core/pull/12972